### PR TITLE
Download plugin custom captions

### PR DESCRIPTION
**Pull request by rica-carv**
Download plugin custom captions & minor code visuals changes

### CODE REVIEW

File: e107_plugins/download/download.php [hunk 1]
```diff
@@ -44,14 +44,15 @@
...
+	if(!defined("e_PAGETITLE")) {define("e_PAGETITLE", LAN_PLUGIN_DOWNLOAD_NAME);}
 
 	if(!defined("USER_WIDTH") && !deftrue('BOOTSTRAP')) { define("USER_WIDTH","width:100%"); }
 
 	/* define images */
 
 	if(deftrue('BOOTSTRAP'))
 	{
-		define("IMAGE_DOWNLOAD", (file_exists(THEME."images/download.png") ? THEME."images/download.png" : e_IMAGE."generic/download.png"));
+		define("IMAGE_DOWNLOAD", (file_exists(THEME."images/download.png") ? THEME."images/download.png" : 'icon-download.glyph'));
```
Comment (CaMer0n) [inline e107_plugins/download/download.php]: How does this change relate to custom captions?

Comment (rica-carv) [inline e107_plugins/download/download.php]: Well, it isn't, but since it's a minor ocrrection....

Comment (CaMer0n) [inline e107_plugins/download/download.php]: It's a correction for a bootstrap site, and a missing icon bug for one that isn't.  ;-)

Comment (rica-carv) [inline e107_plugins/download/download.php]: It's not a correction to a bootstrap site, it's a correction to use the bootstrap icon, if bootstrap is defined, and not use a png icon, if bootstrap is defined, as it is now... ;-)

Comment (CaMer0n) [inline e107_plugins/download/download.php]: You're right, I overlooked that. Thanks.

**Pull request merged**